pilot/pkg/networking/util/util.go

```diff
@@ -100,6 +100,11 @@ const (
 
 	// Well-known header names
 	AltSvcHeader = "alt-svc"
+
+	// Well-known metadata exchange EnvoyFilter config name
+	// TODO: Remove these at 1.14.
+	MXName110 = "metadata-exchange-1.10"
```

Comment:
what about tcp-mx?

Reply:
I just assumed that presence of http mx == presence of tcp one. I cannot imagine a case where user intentionally delete http mx but not tcp mx.